bin/single-new-eks-awsnative-observability.ts

```diff
@@ -1,4 +1,4 @@
-import SingleNewEksAWSNativeobservabilityConstruct from '../lib/single-new-eks-awsnative-observability-construct';
+import SingleNewEksAWSNativeobservabilityConstruct from '../lib/single-new-eks-awsnative-observability';
```

Comment:
Why is class name `SingleNewEksAWSNativeobservabilityConstruct` still having Construct ? We want to totally get rid of word Construct with Pattern. Also we dont want -construct to be removed. We would rather want it to be renamed to -pattern.